e2e/test/prerequisites/E2ETestsSetup/e2eTestsArmTemplate.json

```diff
@@ -2,32 +2,85 @@
     "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
     "contentVersion": "1.0.0.0",
     "parameters": {
-        "Region": {
+        "KeyVaultName": {
```

Comment:
Please move all the parameters with a default after all the ones that don't have one.